### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by timgarrels**

### CODE REVIEW

**Comment by martintaraz** [comment]
Good to go
Maybe add Travis CI badge in the same run
@codeZeilen please merge this PR

File: README.md [hunk 1]
```diff
@@ -1 +1,17 @@
...
+
+I just wanna tell you how I'm feeling
+Gotta make you understand
+
+Never gonna give you up
+Never gonna let you down
+Never gonna run around and desert you
+Never gonna make you cry
+Never gonna say goodbye
+Never gonna tell a lie and hurt you
```
Comment (felixrindt) [inline README.md:17]: could you use markdown quotes here?

**Review by felixrindt — CHANGES_REQUESTED** [review]

**Comment by felixrindt** [comment]
this fixes #1

**Comment by EightSQ** [comment]
100/100 would approve.

File: README.md [hunk 2]
```diff
@@ -1 +1,17 @@
...
+
+We're no strangers to love
+You know the rules and so do I
+A full commitment's what I'm thinking of
+You wouldn't get this from any other guy
+
+I just wanna tell you how I'm feeling
+Gotta make you understand
+
+Never gonna give you up
```
Comment (felixrindt) [inline README.md]: image

**Pull request closed without merging**